libraries/AP_BattMonitor/AP_BattMonitor_FuelLevel_Reventech.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AP_BattMonitor_config.h"
+
+#if AP_BATTERY_FUELLEVEL_REVENTECH_ENABLED
+
+#include <AP_HAL/AP_HAL.h>
+#include "AP_BattMonitor_FuelLevel_Reventech.h"
+
+/*
+  "battery" monitor for liquid fuel level systems.
+
+  Output is:
+
+    - voltage is the calibrated fuel level in the tank
+ */
+extern const AP_HAL::HAL& hal;
+
+const AP_Param::GroupInfo AP_BattMonitor_FuelLevel_Reventech::var_info[] = {
+
+    // @Param: FL_PIN
+    // @DisplayName: Fuel level analog pin number
+    // @Description: Analog input pin that fuel level sensor is connected to. Airspeed ports can be used for Analog input. When using analog pin 103, the maximum value of the input in 3.3V.
+    // @Values: -1:Not Used,11:Pixracer,13:Pixhawk ADC4,14:Pixhawk ADC3,15:Pixhawk ADC6/Pixhawk2 ADC,103:Pixhawk SBUS,8: F405 Periph ADC Pin
+    AP_GROUPINFO("FL_PIN", 45, AP_BattMonitor_FuelLevel_Reventech, _pin, 8),
+
+    // @Param: LS800_MAX_LEVEL
```

Comment:
This naming is odd; are the other sensors suffuciently different they'd need different parameters?